mailpoet/lib/Cron/Triggers/WordPress.php

```diff
@@ -80,8 +80,11 @@ public function run() {
   }
 
   private function checkRunInterval() {
-    $lastRunAt = (int)$this->settings->get(self::LAST_RUN_AT_SETTING, 0);
     $runInterval = $this->wp->applyFilters('mailpoet_cron_trigger_wordpress_run_interval', self::RUN_INTERVAL);
+    if ($runInterval === -1) {
+      return true;
```

Comment:
@websupporter This change means that by default `LAST_RUN_AT_SETTING` will never be updated. My first thought was this information could potentially be useful to someone debugging a customer site and its absence might confuse a supporter to think cron doesn't work. However, after thinking more about it, I believe I'm overestimating the importance of this data - after all, we have daemon's data fields such as `updated_at` as a fallback - and performance benefits outweigh possible confusion.